### ISSUE TITLE
Upgrade jQuery version

### ISSUE THREAD

**Issue by jessedobbelaere** [forkcms/forkcms#934]
I was working on a Fork website and had a very strange behaviour with a jquery plugin on IE9 and IE10. After an hour of digging in the code, I decided to upgrade the jQuery version of Fork from v1.8 to v1.11.1 and it solved my problems.

Is there a reason why Fork CMS still uses jQuery 1.8 and not  v1.11.1? Would it break code? Version 1.8 is already two years old. Should I make a PR?

**Comment by yvesvanlaer** [comment]
I'm using Bootstrap 3.X in my Frontend. Bootstrap requires an upgrade to jQuery 1.9.1 or higher ... So I'm also looking forward to see this upgrade :-).

**Comment by tijsverkoyen** [comment]
@jessedobbelaere The upgrade is more important then the color of the autocomplete. So feel free to PR with the color-issue

**Comment by jessedobbelaere** [comment]
@tijsverkoyen I looked into this last week, but when you search with the latest jQuery UI version you only get the autocomplete with pagetitle and not a title **and description** like Fork always had in the search box. I should dig in the code that fetches the line of page content/description and see why it isn't added to the autocomplete.

Should I only update the frontend jQuery UI or also the backend (because of the bootstrap implementation that will probably already have an upgraded jQuery UI version)?
Oh, I tried to create a [jQuery UI theme with components that resemble the Fork backend](http://jqueryui.com/themeroller/#!zThemeParams=5d00000100f205000000000000003d8888d844329a8dfe02723de3e5702531794cd29e6ed19a93500bec10499630a6324472d2f5d22287bfb9d36af418cfc3cdecc976660c315326f871a11eb0b6ac181031339b8262f3572e9de46bef0b5f3cf89fe6424a433125a2ae44d8c029c905b9dd91a6474002f3f43007c31ac4e89329959f1a2b91b3949c90da9dd7dc60e6396488a6e67e6dcf63f4135a1331bf16e6923d0badc56d0a32ddb1ca83c46390f0e240136d45843154907bf8ca9e94efd3ff8e39f72a7a60c9fc0dca1e1073c0d80b8c2d312b3fa724982e1bd27a938add4a931c120530b2cae76641e84759dfd50c695bef663815e9efca92a267f608fa40fa1d6f7aa59426d0665a8492c3c1aa1506093245c0b923257744fe2a9a07b91cfbcda1c8fd20020afb24688230c28c8e027a99b48967b2dbf59e6124aeecec64593596dd0c4ee5ff9222fb8c441601167f7ca82aed5050f9a767816d339625af17eb699e5da544839ca61e078718aad9f875180ff7c97f0946ac079d39acdf666d8d60492a216cace579df78bfcbbade91ba1fc35f4f2dc035551cfceeefa0cb1e07abe52e6c261e447dda5b4eaed610e84fd88d63c6ff9b15f9eba2ac5c165e1dfd33a03448e1b9f0d406cdb45a3234fc5eb6383c08673f2f186876cbb2dd76d2c53927254093e40232186ffef9a505dd), and I got a similar look for the tabs and calendar etc, but I probably should tweak some more. And then I have to do a lot of backend testing before I can PR. Also, what jQuery UI components are actively used in the frontend beside search autocomplete and maybe a datepicker? Can't think of any other

**Comment by WouterSioen** [comment]
I don't think any others are used.
You can also do a pull request and ask for help to test this. We'd really like to have this upgrade soon, so we'd be happy to help testing.

**Comment by jessedobbelaere** [comment]
I added a PR for the upgrading of jquery/jquery UI. This is only the frontend. Not sure what to do with the backend. I tried to create a look-a-like of the backend jquery ui theme in the [themeroller url](http://jqueryui.com/themeroller/#!zThemeParams=5d00000100f205000000000000003d8888d844329a8dfe02723de3e5702531794cd29e6ed19a93500bec10499630a6324472d2f5d22287bfb9d36af418cfc3cdecc976660c315326f871a11eb0b6ac181031339b8262f3572e9de46bef0b5f3cf89fe6424a433125a2ae44d8c029c905b9dd91a6474002f3f43007c31ac4e89329959f1a2b91b3949c90da9dd7dc60e6396488a6e67e6dcf63f4135a1331bf16e6923d0badc56d0a32ddb1ca83c46390f0e240136d45843154907bf8ca9e94efd3ff8e39f72a7a60c9fc0dca1e1073c0d80b8c2d312b3fa724982e1bd27a938add4a931c120530b2cae76641e84759dfd50c695bef663815e9efca92a267f608fa40fa1d6f7aa59426d0665a8492c3c1aa1506093245c0b923257744fe2a9a07b91cfbcda1c8fd20020afb24688230c28c8e027a99b48967b2dbf59e6124aeecec64593596dd0c4ee5ff9222fb8c441601167f7ca82aed5050f9a767816d339625af17eb699e5da544839ca61e078718aad9f875180ff7c97f0946ac079d39acdf666d8d60492a216cace579df78bfcbbade91ba1fc35f4f2dc035551cfceeefa0cb1e07abe52e6c261e447dda5b4eaed610e84fd88d63c6ff9b15f9eba2ac5c165e1dfd33a03448e1b9f0d406cdb45a3234fc5eb6383c08673f2f186876cbb2dd76d2c53927254093e40232186ffef9a505dd) and tabs, datepicker etc seem the same like they are in the backend. Maybe I'll give it a try to upgrade jquery/jquery UI and test everything. Or should I leave the backend for now, and wait till the conversion to bootstrap and/or twig is finished?
